src/H5FDsubfiling/H5FDsubfiling_priv.h

```diff
@@ -21,18 +21,8 @@
 /* Standard Headers */
 /********************/
 
-/* TODO: review headers needed */
-#include <assert.h>
-#include <libgen.h>
 #include <stdatomic.h>
-#include <stdbool.h>
-#include <stdint.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <unistd.h>
-
-#include "mpi.h"
+#include <libgen.h>
```

Comment:
If libgen.h is for basename/dirname, I added H5_ versions to feature/vfd_swmr to paper over the BSD vs GNU differences.

Reply:
Yes, will need to update that eventually.